app/helpers/offers_helper.rb

```diff
@@ -66,10 +66,14 @@ def offer_type_description(offer)
   end
 
   def estimate_display(offer)
+    return unless offer
+
     currency = Money::Currency.new(offer.currency)
     estimate =
       [offer.low_estimate_cents, offer.high_estimate_cents].compact
-        .map { |amt| (amt / currency.subunit_to_unit).round }.join(' - ')
+        .map do |amt|
+        number_with_delimiter(amt / currency.subunit_to_unit.round)
```

Comment:
This didn't show commas for big numbers, it does now.

Reply:
<img width="1107" alt="Screen Shot 2020-12-17 at 5 22 36 PM" src="https://user-images.githubusercontent.com/19369/102550824-81126e80-408c-11eb-9e78-008e89323a32.png">